client/package.json

```diff
@@ -17,7 +17,7 @@
     "material-design-icons-iconfont": "^5.0.1",
     "moment": "^2.27.0",
     "remove-accents": "^0.4.2",
-    "sbc-common-components": "^2.2.1",
+    "sbc-common-components": "^2.3.1",
```

Comment:
please update the version to latest: 2.3.2
which handles the url issues that we had few days ago

Reply:
Absolutely, will do. Thanks!

Will update in next commit.

Reply:
It's now 2.3.3 (if you want to use the version info tooltip in the footer).

Reply:
Can't keep up lol. Will update